amazon_features/steps/amazon.py

```diff
@@ -0,0 +1,45 @@
+from behave import given, when, then
+
+from features.steps.functions import send_input, click, select_items_extract_prices, total_cart_prices
+
+"""......................all the locators......................"""
+
+tags = {"SEARCH BOX": "xpath=//*[@id='twotabsearchtextbox']",
+        "SEARCH": "xpath=//*[@id='nav-search-submit-button']",
+        "RATING": "xpath =(//ul[@data-csa-c-content-id='1318475031']/span)",
```

Comment:
Are you sure these ID will remain same

Reply:
I am not sure I just took as it is easy to find out the locator. suggest me what else I can do

Reply:
try to find out generic XPath to identify the rantings 
use can use **aria-label="3 Stars & Up"**